### DISCUSSION TITLE
v4.3 CONFIGURATION UTILITY

### DISCUSSION THREAD

**Discussion by NDSU-UGPTI-ATAC** [udotdevelopment/ATSPM#194] [Q&A] [answered]
Is an updated version of the CONFIGURATION UTILITY document available yet? 

https://udottraffic.utah.gov/ATSPM/Images/Configuration-Detection%20Type-Log%20Action%20Taken.pdf
This link on UDOT's v4.3 website still shows a 2019 document which is more relevant to v4.2.4 and older

**Comment by avetraffic** [comment, marked as answer]
We are updating it right now and it should be available in the next week or so.
